cmd/experimental/kjobctl/pkg/builder/slurm_builder.go

```diff
@@ -238,6 +239,9 @@ func (b *slurmBuilder) build(ctx context.Context) (runtime.Object, []runtime.Obj
 		gpusPerTask.Add(*number)
 	}
 
+	totalContainers := len(job.Spec.Template.Spec.Containers)
+	memPerNode := quantityToFloat(b.memPerNode)
+	memPerContainer := int64(math.Ceil(memPerNode / float64(totalContainers)))
```

Comment:
```suggestion
	memPerContainer := b.memPerNode.Mul(int64(1 / len(job.Spec.Template.Spec.Containers)))
```

Maybe like this?

Reply:
Not sure that works, I think the result of `int64(1 / len(job.Spec.Template.Spec.Containers))` will always be `0`.

Reply:
On this case maybe like this?

```
limits[corev1.ResourceMemory] = *resource.NewMilliQuantity(b.memPerNode.MilliValue()/int64(len(job.Spec.Template.Spec.Containers)), b.memPerNode.Format)
```

Reply:
Done. Thanks!